components/Article/Codebox/InlineCode.tsx

```diff
@@ -0,0 +1,8 @@
+import styles from './index.module.scss';
+import type { PropsWithChildren } from 'react';
+
+const InlineCode = ({ children }: PropsWithChildren) => (
```

Comment:
Could you change this to `const InlineCode: React.FC<PropsWithChildren> = ({ children }) =>`